packages/orchestration/src/orchestration-api.ts

```diff
@@ -172,11 +172,6 @@ export interface OrchestrationAccountI {
    * @returns void
    */
   transferSteps: (amount: AmountArg, msg: TransferMsg) => Promise<void>;
-  /**
-   * deposit payment from zoe to the account. For remote accounts,
-   * an IBC Transfer will be executed to transfer funds there.
-   */
-  deposit: (payment: Payment<'nat'>) => Promise<void>;
```

Comment:
cc @dtribble  - change to `orchestration-api.ts`

Reply:
cc @mitdralla too

Reply:
Similar story to: https://github.com/Agoric/agoric-sdk/pull/9591#discussion_r1659007324

Reply:
lgtm